dicebot/handlers/message/tldrwl_handler.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+import logging
+
+from dicebot.data.types.message_context import MessageContext
+from dicebot.handlers.message.abstract_handler import AbstractHandler
+
+from tldrwl.summarize import Summarizer
+from tldrwl.exception import TldrwlException
+
+
+TLDRWL_TRIGGERS = (
+    "tldr",
+    "tldw",
+    "tldl",
+    "tldrwl",
+    "tl;dr",
+    "tl;dw",
+    "tl;dl",
+    "tl;drwl",
+    "bro, I don't have time for this shit",
+)
+
+
+class TldrwlHandler(AbstractHandler):
+    """Get a too long, didn't read/write/listen summary"""
+
+    async def should_handle(
+        self,
+        ctx: MessageContext,
+    ) -> bool:
+        return (
+            ctx.message.reference is not None
+            and ctx.message.content.lower() in TLDRWL_TRIGGERS
+        )
+
+    async def handle(
+        self,
+        ctx: MessageContext,
+    ) -> None:
+        # Appease pyright
+        assert ctx.client.user is not None
+
+        reference = ctx.message.reference
+        # I don't think this can happen if should_handle works properly
+        if not reference or not reference.message_id:
+            logging.info("The message reference is missing...")
+            return
```

Comment:
Correct, normally I'd just `assert reference is not None` above to do the same appeasement as the `ctx.client.user` stuff.